conf/messages.cy

```diff
@@ -1240,3 +1240,86 @@ rentalsAndRentARoomIncomeStart.l3 = *Missing Welsh*
+ukRentARoom.expensesIntro.p4.individual = neu
+ukRentARoom.expensesIntro.p4.agent = neu
+ukRentARoom.expensesIntro.p5.individual = Treuliau unigol, lle rydych yn nodi swm ar gyfer pob math o draul ar wahân.
+ukRentARoom.expensesIntro.p5.agent = Treuliau unigol, lle rydych yn nodi swm ar gyfer pob math o draul ar wahân.
+
+claimExpensesOrRRR.title = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.heading.agent = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.heading.individual = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.checkYourAnswersLabel.amount.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.checkYourAnswersLabel.amount.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.checkYourAnswersLabel.type = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.no = Treuliau
+claimExpensesOrRRR.yes = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.legend.individual = Pa un yr hoffech ei hawlio?
+claimExpensesOrRRR.legend.agent = Pa un hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.error.required.individual = Dewiswch os hoffech hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.error.required.agent = Dewiswch os hoffai’ch cleient hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.amount.error.required.individual = Nodwch swm y Rhyddhad Rhentu Ystafell yr hoffech ei hawlio
+claimExpensesOrRRR.amount.error.required.agent = Nodwch swm y Rhyddhad Rhentu Ystafell yr hoffai’ch cleient ei hawlio
+claimExpensesOrRRR.amount.error.nonNumeric.individual = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.nonNumeric.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.individual = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.outOfRange = Nodwch swm rhwng ¬£{0} a ¬£{1}
+claimExpensesOrRRR.change.hidden = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.rentARoom = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.expenses = Treuliau
+claimExpensesOrRRR.radio.question.label.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.radio.question.label.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.para1.jointlyLet.individual = Gallwch hawlio hyd at £{0} y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
```

Comment:
`claimExpensesOrRRR.para1.jointlyLet.agent ` ? Pls test again the files you have changed